### PR TITLE
Update runtime example

### LINKED ISSUE #258: Example doesn't match the spec doc in for state.json
In `specs/runtime.md`, the example of a `state.json` file does not match the text immediately above it in the following ways:

* there is no `version` property
* the `bundlePath` property is called `root` in the example

### PR DESCRIPTION
**Pull request by marcosnils**
Fixes #258

### CODE REVIEW

File: runtime.md [hunk 1]
```diff
@@ -27,9 +27,10 @@ This is provided so that consumers can find the container's configuration and ro
 
 ```json
 {
+    "version": "0.2",
```
Comment (wking) [inline runtime.md]: This is handled by #144, which I've just rebased to reflect the current 0.2.0 version.

File: runtime.md [hunk 2]
```diff
@@ -27,9 +27,10 @@ This is provided so that consumers can find the container's configuration and ro
...
     "id": "oc-container",
     "pid": 4422,
-    "root": "/containers/redis"
+    "bundlePath": "/containers/redis"
```
Comment (wking) [inline runtime.md]: This is handled by #257.

**Comment by wking** [comment]
I think both of the changes you're making are handled by existing PRs,
so I suggest we close this as a duplicate (and get those other PRs
landed :).

**Comment by marcosnils** [comment]
Duplicate of #257 and #144

**Pull request closed without merging**